cmd/lava/internal/help/helpdoc.go

```diff
@@ -282,3 +284,89 @@ General-purpose environment variables:
 		considered experimental.
 	`,
 }
+
+// HelpChecktypes documents the Lava checktypes file.
+var HelpChecktypes = &base.Command{
+	UsageLine: "checktypes",
+	Short:     "checktype catalog file",
```

Comment:
I made the error of using the word `file` here when, in fact, a catalog can be a remote resource.

```suggestion
	Short:     "checktype catalog format",
```